uaclient/apt.py

```diff
@@ -259,63 +273,56 @@ def get_esm_cache():
         apt_pkg.init()
         # If the rootdir folder doesn't contain any apt source info, the
         # cache will be empty
-        cache = apt.Cache(rootdir=ESM_APT_ROOTDIR)
+        # If the structure in the rootdir folder does not exist or is
+        # incorrect, an exception will be raised
+        return apt_pkg.Cache(None)
```

Comment:
I think I am missing something, how we are getting her only the esm cache ?

Reply:
Because we are setting the `apt_pkg` configuration key right above. When we create the cache like this it will pick up whatever is in the config